packages/lit-dev-content/site/docs/v3/components/decorators.md

```diff
@@ -44,38 +32,27 @@ The `@customElement` decorator defines a custom element, equivalent to calling:
-| {% api-v3 "@queryAssignedElements" "queryAssignedElements" %} | Defines a property that returns the child elements assigned to a specific slot. | [Shadow DOM](/docs/v3/components/shadow-dom/#query-assigned-nodes) |
-| {% api-v3 "@queryAssignedNodes" "queryAssignedNodes" %} | Defines a property that returns the child nodes assigned to a specific slot. | [Shadow DOM](/docs/v3/components/shadow-dom/#query-assigned-nodes) |
+Decorators are a [stage 3 proposal](https://github.com/tc39/proposal-decorators) for addition to the ECMAScript standard. Compilers like [Babel](https://babeljs.io/) and [TypeScript](https://www.typescriptlang.org/) support decorators, though no browsers have implemented them yet. Lit decorators work with Babel and TypeScript, and will work in browsers when they implement them natively.
 
-## Importing decorators
+{% aside "info" %}
 
-You can import all the lit decorators via the `lit/decorators.js` module:
+What does stage 3 mean?
 
-```js
-import {customElement, property, eventOptions, query} from 'lit/decorators.js';
-```
+It means that the specification text is complete, and ready for browsers to implement. Once the specification has been implemented in multiple browsers, it can move to the final stage, stage 4, and be added to the ECMAScript standard. A stage 3 proposal will only change if critical issues are discovered during implementation.
 
-To reduce the amount of code needed to run the component, decorators can be imported individually into component code. All decorators are available at `lit/decorators/<decorator-name>.js`. For example,
+{% endaside %}
 
-```js
-import {customElement} from 'lit/decorators/custom-element.js';
-import {eventOptions} from 'lit/decorators/event-options.js';
-```
+Previous versions of the decorators proposal are also supported by some compilers, most notably TypeScript's "experimental decorators" which Lit has supported since its inception.
+
+Lit decorators work as both TypeScript experimental decorators _and_ standard decorators meaning they work with either configuration set in tsconfig. However, we currently recommend that TypeScript users use experimental decorator mode, i.e. projects should set `"experimentalDecorators": true`, because the emitted JavaScript is much smaller than in standard decorator mode. As such, all of the TypeScript code samples in our documentation reflect the syntax that works with this mode – namely, we omit the `accessor` keyword on class fields that's required for [standard decorators](#standard-decorators).
+
+If you are working in a TypeScript project that requires `"experimentalDecorators": false` or `useDefineForClassFields: true`, consult the [Migrating TypeScript to standard decorators](#migrating-typescript-standard-decorators) section of this page.
 
 ## Enabling decorators { #enabling-decorators }
```

Comment:
I think we should consider nesting this and *Importing Decorators* under a *Using Decorators* heading